python/tvm/relay/op/contrib/ethosn.py

```diff
@@ -64,14 +64,42 @@ def ConvertEquivalents() -> tvm.ir.IRModule:  # pylint: disable=invalid-name
     """Converts operations into a numerically equivalent form
     that can be understood by the NPU codegen.
 
-    Return
-    ------
+    Returns
+    -------
     Pass
         The module pass.
     """
     return _ethosn.ConvertEquivalents()
 
 
+def InlineNonComputeIntensivePartitions() -> tvm.ir.IRModule:  # pylint: disable=invalid-name
```

Comment:
nit: if possible, avoid ignoring pylint

Reply:
The name is like this to simply follow the style of other pass names, in general though I agree

Reply:
oh yeah I missed that as the pylint pragmas were at the top of those files at other places.